src/components/AddEditForm.js

```diff
@@ -89,12 +50,12 @@ const AddForm = ({onAdd,open,setOpen}) => {
                 component='p' 
                 textAlign='center'>
                 <Box>
-                  <label>Add content of note</label> <br />
+                  {indexEditNote > -1 ? 'Edit content of note' : 'Add content of note'}<br />
```

Comment:
can we use something more appropriate instead of -1, for example, we can identify if we are in "edit mode" if we have a note with an id? And then we don't have to work with index - 1, etc.